### PR TITLE
Apply ktlint formatting

### PR DESCRIPTION
**Pull request by zigavehovec**
## Description

Kotlin code isn't properly formatted.

## Changes

This PR applies ktlint formatting introduced in https://github.com/bitmovin/bitmovin-player-react-native/pull/276.

## Checklist
- [ ] 🗒 `CHANGELOG` entry

### CODE REVIEW

**Review by rolandkakonyi — APPROVED** [review]
I see the check passed so approving.
I can't say much about the actual changes :)

File: android/src/main/java/com/bitmovin/player/reactnative/OfflineModule.kt [hunk 1]
```diff
@@ -260,20 +269,24 @@ class OfflineModule(private val context: ReactApplicationContext) : ReactContext
...
-                ?: promise.reject(IllegalArgumentException("Could not find the offline module instance"))
+            ?: promise.reject(IllegalArgumentException("Could not find the offline module instance"))
     }
 
     /**
      * Helper function that returns the initialized `DrmModule` instance.
      */
     private fun drmModule(): DrmModule? =
-            context.getNativeModule(DrmModule::class.java)
+        context.getNativeModule(DrmModule::class.java)
```
Comment (matamegger) [inline android/src/main/java/com/bitmovin/player/reactnative/OfflineModule.kt:285]: NIT: Would fit into one line I think.

**Review by matamegger — APPROVED** [review]
Well it is an automation, so there shouldn't be a lot to add.
From my side this is fine to be merged as it is right. See my comment's as NITs. At least one of them is probably not "fixable" anyway.

File: android/src/main/java/com/bitmovin/player/reactnative/OfflineModule.kt [hunk 2]
```diff
@@ -260,20 +269,24 @@ class OfflineModule(private val context: ReactApplicationContext) : ReactContext
...
 
     /**
      * Helper function that returns the initialized `UIManager` instance.
      */
     private fun uiManager(): UIManagerModule? =
-            context.getNativeModule(UIManagerModule::class.java)
+        context.getNativeModule(UIManagerModule::class.java)
```
[the remark this answers is not in the document]
Comment (zigavehovec) [reply android/src/main/java/com/bitmovin/player/reactnative/OfflineModule.kt]: Good point, I missed that.
https://github.com/bitmovin/bitmovin-player-react-native/pull/279/commits/bc9fa39946806d795b91fc352d36d9c3728ccead

**Pull request merged**